assets/js/script.js

```diff
@@ -117,4 +117,10 @@ document.addEventListener("DOMContentLoaded", function () {
             popover.classList.remove('popover--active');
         }
     });
+
+    // show popover on first visit
+    if (!localStorage.getItem('isReturningVisitor')) {
```

Comment:
localStorage feature detect brauchts noch

Reply:
Das bläht den Code so auf. Willste wirklich? Technisch ist es in allen gängigen Browsern verfügbar, und wer es deaktiviert oder ernsthaft nicht hat, sieht halt das Popover immer wieder.

Reply:
Das gibt hier sonst nen syntax error oder?

Reply:
hab mal das nötigste ergänzt.. ist nicht soo viel find ich